project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -165,6 +165,16 @@ class PreferenceForm extends React.Component {
           restaurants = restaurants.concat(restaurant.results);
         }
 
+        if (restaurants.length === 0) {
+          let message = "We can't find any restaurants near you";
+          if (this.state.open) {
+            message += ' that are currently open';
+          }
+          message += '. Please try changing your preferences.';
+          alert(message);
```

Comment:
possibly for a follow up- but I still think a pop-up alert could be too jarring, maybe just displaying this message at the place where we would show the results if there were any?

Reply:
you could just create a NoResults component that would be rendered in this case